docs/sphinx/examples/python/tutorials/vqe_water_active_space.ipynb

```diff
@@ -0,0 +1,218 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Water molecule with active space (CPU vs. GPU)\n",
+    "\n",
+    "#### A- Classical simulation as a reference: CCSD"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "converged SCF energy = -75.9839755372789\n",
+      "Total number of electrons=  10\n",
+      "Total number of orbitals=  13\n",
+      "ncore occupied orbitals=  3\n",
+      "E(CCSD) = -75.98508980454677  E_corr = -0.001114267267875565\n",
+      "Total CCSD energy for active space Hamiltonian=  -75.98508980454677 \n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "# pip install pyscf\n",
+    "\n",
+    "from pyscf import gto, scf, mcscf, cc\n",
+    "\n",
+    "geometry='O 0.1173 0.0 0.0; H -0.4691 0.7570 0.0; H -0.4691 -0.7570 0.0'\n",
+    "mol=gto.M(\n",
+    "    atom=geometry,\n",
+    "    spin=0,\n",
+    "    charge=0,\n",
+    "    basis='631g',\n",
+    ")\n",
+    "\n",
+    "myhf=scf.RHF(mol)\n",
+    "myhf.max_cycle=100\n",
+    "myhf.kernel()\n",
+    "nelec = mol.nelectron\n",
+    "print('Total number of electrons= ', nelec)\n",
+    "norb = myhf.mo_coeff.shape[1]\n",
+    "print('Total number of orbitals= ', norb)\n",
+    "\n",
+    "norb_cas, nele_cas = (4,4)\n",
+    "mycasci = mcscf.CASCI(myhf, norb_cas, nele_cas)\n",
+    "print('ncore occupied orbitals= ', mycasci.ncore)\n",
+    "\n",
+    "frozen=[]\n",
+    "frozen+=[y for y in range(0,mycasci.ncore)]\n",
+    "frozen+=[y for y in range(mycasci.ncore+norb_cas, len(mycasci.mo_coeff))]\n",
+    "mycc=cc.CCSD(myhf,frozen=frozen).run()\n",
+    "print('Total CCSD energy for active space Hamiltonian= ', mycc.e_tot, '\\n')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### B- VQE-UCCSD:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/tmp/ipykernel_123972/4288414001.py:36: ComplexWarning: Casting complex values to real discards the imaginary part\n",
```

Comment:
Do you know what is causing this warning? It would be nice to get rid of it if possible.

Reply:
Yes. The hamiltonian from openfermion is exported as real number and we should convert this into complex number. I will modify it.

Reply:
The problem here comes from the data type in qubit hamiltonian. 
[/tmp/ipykernel_317959/2875759736.py:39](https://vscode-remote+attached-002dcontainer-002b7b22636f6e7461696e65724e616d65223a222f6d66617261672d62656e63686d61726b227d-0040ssh-002dremote-002b10-002e20-002e100-002e80.vscode-resource.vscode-cdn.net/tmp/ipykernel_317959/2875759736.py:39): ComplexWarning: Casting complex values to real discards the imaginary part
  spin_ham=cudaq.SpinOperator(qubit_hamiltonian)

The coefficient in qubit hamiltonian are complex values. It looks like cudaq.SpinOperator is converting this into real values. I am not sure how to change this here. This issue should not affect the problem here because all the imaginary values are zeros.